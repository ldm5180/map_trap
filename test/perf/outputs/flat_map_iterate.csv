Run on (4 X 2594.09 MHz CPU s)
2017-10-28 17:23:48
name,iterations,real_time,cpu_time,time_unit,bytes_per_second,items_per_second,label,error_occurred,error_message
"BM_map_iterate<boost::container::flat_map<int64_t, int64_t>>/8",15459024,44.527,44.5044,ns,,,,,
"BM_map_iterate<boost::container::flat_map<int64_t, int64_t>>/16",5854030,120.866,120.864,ns,,,,,
"BM_map_iterate<boost::container::flat_map<int64_t, int64_t>>/32",1656923,425.887,425.684,ns,,,,,
"BM_map_iterate<boost::container::flat_map<int64_t, int64_t>>/64",388108,1765.79,1765.77,ns,,,,,
"BM_map_iterate<boost::container::flat_map<int64_t, int64_t>>/128",87346,8279.49,8275.62,ns,,,,,
