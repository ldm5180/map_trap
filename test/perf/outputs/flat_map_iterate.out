Run on (4 X 2594.09 MHz CPU s)
2017-10-28 16:29:11
name,iterations,real_time,cpu_time,time_unit,bytes_per_second,items_per_second,label,error_occurred,error_message
"BM_map_iterate<boost::container::flat_map<int, std::string>>/8",1059887,680.134,680.425,ns,,,,,
"BM_map_iterate<boost::container::flat_map<int, std::string>>/16",267518,2621.41,2622.7,ns,,,,,
"BM_map_iterate<boost::container::flat_map<int, std::string>>/32",63908,10718.8,10714.1,ns,,,,,
"BM_map_iterate<boost::container::flat_map<int, std::string>>/64",14641,48058.6,48076.7,ns,,,,,
"BM_map_iterate<boost::container::flat_map<int, std::string>>/128",3954,178119,178206,ns,,,,,
